src/Data/TableRowsAsRecordsDataProvider.php

```diff
@@ -268,47 +268,47 @@ public function save(ModelInterface $objItem, $recursive = false)
             }
 
             if ($intId > 0) {
-                $this->objDatabase
-                    ->prepare(sprintf('UPDATE %s %%s WHERE id=? AND %s=?', $this->strSource, $this->strGroupCol))
+                $this->connection
+                    ->prepare(sprintf('UPDATE %s %%s WHERE id=? AND %s=?', $this->source, $this->strGroupCol))
                     ->set($arrSQL)
```

Comment:
This shouldn't work. Afaik there's no set method using doctrine dbal. The `%s` is also wrong.

Reply:
Has been fixed